docs/_includes/sidebar.html

```diff
@@ -1,46 +1,45 @@
-{% comment %}
-    Create an empty array to store secondary nav sections
-{% endcomment %}
-{% assign secondary_sections = '' | split: ',' %}
-
-{% for p in site.pages %}
-    {% if p.section == page.section and p.secondary_section %}
-        {% unless secondary_sections contains p.secondary_section %}
-            {% assign secondary_sections = secondary_sections | push: p.secondary_section %}
-        {% endunless %}
-    {% endif %}
-{% endfor %}
-
-{% comment %}
-    Alphabetize the sections
-{% endcomment %}
-{% assign secondary_sections = secondary_sections | sort %}
-
-{% if secondary_sections %}
 <ul class="m-list m-list__unstyled">
-    {% for secondary_section in secondary_sections | sort %}
+{% for first in site.data.side-navigation.first-level %}
+    {% assign first_level_slug = first.heading | slugify %}
     <li class="m-list_item u-mb15">
-        <span class="h5">{{ secondary_section }}</span>
-        <ul class="m-list m-list__links">
-            {% for p in site.pages %}
-                {% if p.section == page.section and p.secondary_section == secondary_section %}
+        <div class="h4">
+            <a class="m-list_link" href="{{ first_level_slug | relative_url }}">{{ first.heading }}</a>
+        </div>
+        {% if page.section == first_level_slug or page == blank %}
+        <div class="m-list_item__tertiary">
+            {% if first.nav-items != '' %}
```

Comment:
I think this check needs to be more robust. I'm seeing this show up empty on the components page.